google-cloud-logging/README.md

```diff
@@ -44,49 +59,83 @@ entry.resource.labels[:version_id] = "20150925t173233"
 logging.write_entries entry
 ```
 
-## Rails Integration
+## Logging in Rack-based frameworks
+
+The `google-cloud-logging` library provides framework integration for popular 
+Rack-based frameworks, such as Ruby on Rails and Sinatra, which sets the default 
+Rack logger to an instance of Stackdriver Logging logger.
 
-This library also provides a built in Railtie for Ruby on Rails integration. When enabled, it sets an instance of Google::Cloud::Logging::Logger as the default Rails logger. Then all consequent log entries will be submitted to the Stackdriver Logging service.
+If you're using Ruby on Rails and the `stackdriver` gem, bundler automatically 
+loads the library into your application when it starts.
+
```

Comment:
What about Rails and `google-cloud-logging`? Document `require "google/cloud/logging/rails"`.

Also see comment on Debugging.